components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -319,16 +321,22 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
 
   // Create a  Kubeflow deployment.
   private async _createDeployment() {
-    for (const prop of ['project', 'zone', 'ipName', 'deploymentName', 'hostName']) {
+    for (const prop of ['project', 'zone', 'deploymentName']) {
       if (this.state[prop] === '') {
         this.setState({
-          dialogBody: 'All fields are required, but it looks like you missed something.',
+          dialogBody: 'Some required fields (project, zone, deploymentName) are missing',
           dialogTitle: 'Missing field',
         });
         return;
       }
     }
 
+    if (this.state.hostName === '') {
+      this.setState({
+        hostName: this.state.deploymentName + '.endpoints.' + this.state.project + '.cloud.goog',
+      });
+    }
+
```

Comment:
Does this mean that if a user wants they can specify a completely different hostname?

Reply:
Yes, currently we only support the default one.
We can add support to custom hostname later.

Reply:
We can remove hostname field from UI and print it in logs?